spec/models/requests/aeon_url_spec.rb

```diff
@@ -0,0 +1,92 @@
+                                           "items" => [{ "holding_id" => "22667098990006421", "id" => "23667098920006421", "status_at_load" => "1", "enumeration" => "Vol. 4: no. 1 - 6", "barcode" => "32101091127959", "copy_number" => "1" }, { "holding_id" => "22667098990006421", "id" => "23667098930006421", "status_at_load" => "1", "enumeration" => "Vol. 5: no. 1-6", "barcode" => "32101094417795", "copy_number" => "1" }, { "holding_id" => "22667098990006421", "id" => "23667098940006421", "status_at_load" => "1", "enumeration" => "Vol. 3: no. 1 - 6", "barcode" => "32101071686446", "copy_number" => "1" },
+                                                       { "holding_id" => "22667098990006421", "id" => "23667098950006421", "status_at_load" => "1", "enumeration" => "Vol. 2: no. 1 - 6", "barcode" => "32101071686438", "copy_number" => "1" }, { "holding_id" => "22667098990006421", "id" => "23667098970006421", "status_at_load" => "0", "enumeration" => "9", "barcode" => "ISSitm51830.290-princetondb" }, { "holding_id" => "22667098990006421", "id" => "23667098960006421", "status_at_load" => "1", "enumeration" => "Vol 1: no. 1 - 4", "barcode" => "32101071302192", "copy_number" => "1" }] } }
+      described_class.new(document:, holding:).to_s
+    end
+    it('takes the call number from the holding') do
+      expect(subject).to include('CallNumber=153521+Pams+%2F+NR+%2F+Chinese+%2F+Box+41')
+    end
+  end
+  context 'when a specific item is passed in' do
+    subject do
+      item = Requests::Requestable::Item.new({ "barcode" => "32101071302192", "id" => "23667098960006421", "holding_id" => "22667098990006421", "copy_number" => "1", "status" => "Available", "status_label" => "Item in place", "status_source" => "base_status", "process_type" => nil, "on_reserve" => "N", "item_type" => "Closed", "pickup_location_id" => "rare", "pickup_location_code" => "rare", "location" => "rare$ctsn", "label" => "Special Collections - Cotsen Children's Library", "description" => "Vol 1: no. 1 - 4 Jan 2013 - Oct 2013", "enum_display" => "Vol 1: no. 1 - 4", "chron_display" => "Jan 2013 - Oct 2013", "in_temp_library" => false })
+      described_class.new(document:, item:).to_s
+    end
+    it('takes the barcode from the item') do
+      expect(subject).to include('ItemNumber=32101071302192')
+    end
+    it('takes enumeration from the item') do
+      expect(subject).to include('rft.volume=Vol+1%3A+no.+1+-+4')
```

Comment:
Per Aeon's OpenURL mapping table it should accept "rft.volume" as a valid param for volume info, however by manual test it does not. It did accept just plain "volume". The Aeon form param is literally "ItemVolume", maybe for now we should pass enumeration as that param.